sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -35,59 +37,66 @@ object Webpack {
     * @return The written file
     */
   def writeConfigFile(
-    emitSourceMaps: Boolean,
-    webpackEntryPoints: Seq[(String, File)],
-    targetDir: File,
-    log: Logger
+      emitSourceMaps: Boolean,
+      webpackEntryPoints: Seq[(String, File)],
+      targetDir: File,
+      log: Logger
```

Comment:
Sorry about that, I'd been avoid squashing since some folks were tracking the branch and things were in flux a lot. I've pulled out the stuff that makes sense as separate PR's and squashed the remainder of this branch down aside from the hard dependencies on the other PR's.